include/ensmallen_bits/de/de_impl.hpp

```diff
@@ -75,21 +92,21 @@ inline double DE::Optimize(DecomposableFunctionType& function,
         l = arma::as_scalar(arma::randi<arma::uvec>(
```

Comment:
@rcurtin - as of Armadillo 8.400, it should be possible to use a simpler form instead of generating a temporary vector: `l = arma::randi<uword>( arma::distr_param(0, populationSize - 1) );`

Ubuntu 18.04 (current LTS) has Armadillo 8.400, while Fedora has more recent versions.  It might be worthwhile to bump the minimum required version to 8.400, as that brings in more functionality compared to 6.500. This includes the many improvements in sparse matrices.